lettuce/flows/__init__.py

```diff
@@ -6,6 +6,7 @@
 from lettuce.flows.couette import CouetteFlow2D
 from lettuce.flows.poiseuille import PoiseuilleFlow2D
 from lettuce.flows.doublyshear import DoublyPeriodicShear2D
+from lettuce.flows.decayingturbulence import DecayingTurbulence2D, DecayingTurbulence3D
```

Comment:
Maybe add them as "decaying2D" and "decaying3D" to the `flow_by_name` dictionary